### ISSUE TITLE
OS Viewer: Retrieve translations from the API

### ISSUE THREAD

**Issue by larjohn** [openspending/openspending#1156]
Is it currently possible to retrieve the translations of dimension/measures labels from the API? If I am not mistaken they are currently retrieved from https://github.com/openspending/os-viewer/blob/master/app/config/translations.json but this is kind of "hardcoded" for our needs in OpenBudgets.eu

Do you have some alternatives on mind about this?

**Comment by akariv** [comment]
This file is used for translating the UI strings, not the data itself. Is
that what you need?

**Comment by larjohn** [comment]
No, I mean the labels that are displayed in the left part of the UI, for instance in the drilldown section. Those are the labels of the dimensions not the labels of the values of those dimensions. The labels of dimensions vary with them. In OBEU we can have a new set of dimensions in each data structure definition, which comes with translations that can be displayed in the UI. The way it works right now, we would either need to manually update the translations json or somehow make it to redirect somewhere on the API, where a custom endpoint would provide a more current file.

**Comment by akariv** [comment]
The names of the hierarchies are retrieved from the datatypes, and not from the data itself.
e.g. all dimensions with a 'functional-classification:*' type will be under the 'Functional Classification' hierarchy, regardless of the titles in the data.
